pkg/master/analyze.go

```diff
@@ -136,6 +136,8 @@ func (phase *serverPhaseAnalysis) SendNodes(in *service.AnalysisMessage) (*servi
 
 	if in.PackageNode != nil {
 		log.Printf("Connecting package node %v to targets.", in.PackageNode.Name)
+		// prevent inserting metadata twice
+		in.PackageNode.AdditionalInfo = nil
```

Comment:
We should be able to add more infonodes otherwise this would be useless

Reply:
The additional info nodes are already inserted in db from the root package node. In the analysis we didn't add new additional info nodes. In the future if we change the analysis step and add more info nodes we can change that.

Reply:
It is the purpose of this method to add new infonodes to the package node.